js/src/Ice/ConnectionI.js

```diff
@@ -1154,6 +1176,10 @@ export class ConnectionI {
                 let message = this._sendStreams.shift();
                 this._writeStream.swap(message.stream);
                 message.sent();
+                if (message.receivedReply) {
+                    completed ??= [];
+                    completed.push(message);
+                }
```

Comment:
Should the break be return like there others, or make the others break so that thy hit the assert check.